### ISSUE TITLE
LOGRO : Conocimiento Superior.

### ISSUE THREAD

**Issue by Alfonn** [RoDNet/Bugtracker-Infinium-4.0.6a#1458]
Bien empecé a realizar dicho logro ''Conocimiento superior'' el cual requiere que encuentres y leas los libros específicos que andan escondidos por Dalaran. Encontré sólo 1 de ellos lo leí y no me apareció como leido en el logro. Todos los demás libros no se encuentran en sus respectivas posiciones. ¿Podrían arreglarlo?

WowHead : http://es.wowhead.com/achievement=1956/conocimiento-superior

**Comment by ghost** [comment]
Yo lo había completado, y todos los libros estan donde deben, solo que algunos estan "ocultos" en el piso, en vez de estar arriba en un estante, estan en el piso y no se ven.

**Comment by Wiinter** [comment]
Yo tambien lo he completado y estan todos los libros, busca bien

**Comment by DatJetto** [comment]
No, los libros no están, yo fui con él por que yo también lo había conseguido y sabía donde estaban, tras buscar un buen rato, no estaba ninguno, ni los que están a la vista ni los escondidos.

**Comment by Wiinter** [comment]
Pues han desaparecido como los portales a Tierras Devastadas de la Horda
